crates/oxc_linter/src/lib.rs

```diff
@@ -142,6 +142,24 @@ impl Linter {
         rules.into_iter().flat_map(|(_, ctx)| ctx.into_message()).collect::<Vec<_>>()
     }
 
+    /// # Panics
+    pub fn print_config<W: Write>(&self, writer: &mut W) {
+        // merge rules with eslint_config
+        let additional_rules = OxlintRules::from_rules(self.rules.clone());
+        let rules = serde_json::to_value(additional_rules).unwrap();
+        let mut config = serde_json::to_value(self.eslint_config.as_ref()).unwrap();
+        config
+            .get_mut("rules")
+            .unwrap()
```

Comment:
Please use `expect` and provide an error message, otherwise users will have no idea what happened.